app/templates/view_buyers.html

```diff
@@ -55,7 +55,7 @@
 {% if brief %}
     <div>
         <hr/>
-        <h3>Brief Details</h3>
+        <h3>Opportunity Details</h3>
```

Comment:
There are still some instances of briefs mentioned in this file. Check line 29 and 31 for example.

Reply:
fixed line 29 and 31 line